commerce_coordinator/apps/rollout/pipeline.py

```diff
@@ -47,7 +48,7 @@ def run_filter(self, request):
 
         if is_redirect_to_commercetools_enabled_for_user(request) and commercetools_available_course:
             active_order_management_system = COMMERCETOOLS_FRONTEND
-        elif sku:
+        elif sku and is_user_enterprise_learner(request):
             active_order_management_system = FRONTEND_APP_PAYMENT_CHECKOUT
```

Comment:
We want to disallow enterprise learners to purchase via CT. The current implementation logic is doing otherwise. It is not checking for enterprise learners in the condition on which we are setting CT as the active order management system.

The logic should be like:

```  
if is_redirect_to_commercetools_enabled_for_user(request) and commercetools_available_course and not is_user_enterprise_learner(request):
    active_order_management_system = COMMERCETOOLS_FRONTEND
elif sku:
    active_order_management_system = FRONTEND_APP_PAYMENT_CHECKOUT
```

Reply:
@shafqatfarhan I can add these changes as part of [SONIC-251](https://2u-internal.atlassian.net/browse/SONIC-251)